### ISSUE TITLE
AzureResourceManagerTemplateDeploymentV3 auto parses output?

### ISSUE THREAD

**Issue by svrooij** [MicrosoftDocs/azure-devops-docs#12277]
In the log of this task I found the following message: 

`Updated output variable '{deploymentOutputs}.{arm_output_name}.type', which contains the outputs section of the current deployment object in string format.`

This suggests that outputs are parsed automatically? I also found this code which suggests the same: https://github.com/microsoft/azure-pipelines-tasks/blob/88e80261f6b825f27e98c5fcc7c877accfb5d1b5/Tasks/AzureResourceManagerTemplateDeploymentV3/operations/DeploymentScopeBase.ts#L74-L91

The documentation for `deploymentOutputs` should be updated accordingly, since manual parsing is no longer needed.

There still isn't a good way to do this for securestring outputs, but it's a start.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3aa4bb5b-30b2-bb87-5a4d-d1fd4f8b9860
* Version Independent ID: aba4f40c-5b1d-404e-7594-aa8887ad5ce5
* Content: [Azure Resource Group Deployment task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-resource-group-deployment?view=azure-devops)
* Content Source: [docs/pipelines/tasks/deploy/azure-resource-group-deployment.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/main/docs/pipelines/tasks/deploy/azure-resource-group-deployment.md)
* Product: **devops**
* Technology: **devops-cicd-tasks**
* GitHub Login: @RoopeshNair
* Microsoft Alias: **ronai**

**Comment by WilliamAntonRohm** [comment]
@svrooij -- Stephan, thank you for your feedback and contribution.

@RoopeshNair -- Roopesh, please look into this issue.

**Comment by svrooij** [comment]
@WilliamAntonRohm I'm not sure how the docs on this part are updated, but I updated the documentation in the task, I'm guessing that flows through to the documentation page automatically?

**Comment by WilliamAntonRohm** [comment]
@svrooij -- Stephan, once your new PR is merged, this issue will be auto-closed.  The docs.microsoft.com article will still need to be updated separately.

@RoopeshNair -- Roopesh, if you'll give me the updated wording for the description of `deploymentOutputs` in the [Arguments](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/deploy/azure-resource-group-deployment?view=azure-devops#arguments) section, I'll create a doc update PR for it.

**Comment by svrooij** [comment]
@WilliamAntonRohm I cannot get a response on the PR that fixes the bug in the task. How do I get feedback on that PR it's been open for almost two months now.
